index.js

```diff
@@ -48,6 +48,7 @@ function send(ctx, path, opts) {
     var maxage = opts.maxage || opts.maxAge || 0;
     var hidden = opts.hidden || false;
     var format = opts.format === false ? false : true;
+    var extensions = Array.isArray(opts.extensions) ? opts.extensions : false;
```

Comment:
Hmmm so what about adding type checker to existing loop? It may look quite '🐟' in code but would preserve complexity.